onnx/test/shape_inference_test.py

```diff
@@ -2989,11 +2997,38 @@ def test_einsum_sum_along_dim(self):  # type: () -> None
 
     def test_einsum_ellipsis(self):  # type: () -> None
         graph = self._make_graph(
-            [('x', TensorProto.FLOAT, (3, 4))],
+            [('x', TensorProto.FLOAT, (3, 4, 4))],
             [make_node('Einsum', ['x'], ['y'], equation='... ii ->... i')],
             [],)
         self._assert_inferred(graph, [make_tensor_value_info('y', TensorProto.FLOAT, (None, None))])  # type: ignore
 
+    def test_einsum_ellipsis_2(self):  # type: () -> None
```

Comment:
Can we add an "implicit" version of this test too ? Something like "...ij,...jk" alone ?

Reply:
Sure, I added this test.